Assets/MixedRealityToolkit.Services/InputSystem/NearInteractionTouchable.cs

```diff
@@ -129,6 +129,32 @@ public void SetBounds(Vector2 newBounds)
             bounds = newBounds;
         }
 
+        /// <summary>
+        /// Adjust the bounds, local center and local forward to match a given box collider
+        /// </summary>
+        public void SetTouchableCollider(BoxCollider newCollider)
```

Comment:
hey while here, can you add doccomments to all the public fields in this file? some are here and some are missing.
for override you can do /// < inheritdoc / >

Reply:
tooltips are missing as well :-) . If you're already adding the /// please add tooltips <3

Reply:
@CDiaz-MS  reminder to add doccomments to all public fields in this file.